pkg/scalers/splunk_observability_scaler.go

```diff
@@ -0,0 +1,183 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"regexp"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/signalfx/signalflow-client-go/v2/signalflow"
+	v2 "k8s.io/api/autoscaling/v2"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+
+	"github.com/kedacore/keda/v2/pkg/scalers/scalersconfig"
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+type splunkObservabilityMetadata struct {
```

Comment:
Should we support a parameter to override an endpoint in case that changes in the future?

Reply:
Note to self: TODO base url overwriting

Reply:
do we expect the endpoint to be changed? if so we can add the parameter.

Reply:
I am not interacting with the endpoint url directly, I am using this wrapper: https://github.com/signalfx/signalflow-client-go . I hope that when base url and endpoint change in the future, this wrapper gets updated.